standards/8x.md

```diff
@@ -0,0 +1,111 @@
+
+Rather than instantiating that specific contract, it simply instantiates a collectible interface with a particular address. We believe other developers should be able to do this for their applications too.
+
+## Motivation
+We want to set a standard for interacting with Layer 2 protocols, not how those protocols operate.
+
+#### Public View Functions
+
+###### isValidSubscription
+```SOLIDITY
+
+/** @dev Checks if the subscription is valid.
+  * @param _subscription is the identifier of the customer's subscription with its relevant details.
+  * @return success is the result of whether the subscription is valid or not.
+*/
+
+function isValidSubscription(bytes32 _subscription)
+    public
+    view
+    returns (bool success);
+```
+
+###### getSubscriptionTokenAddress
+```SOLIDITY
+/** @dev Gets the token contract address within the subscription.
+  * @param _subscription is the identifier of the customer's subscription with its relevant details.
+  * @return subscriptionTokenAddress is token contract address within the subscription.
+*/
+function getSubscriptionTokenAddress(bytes32 _subscription)
+    public
+    view
+    returns (address subscriptionTokenAddress);
+```
+
+###### getSubscriptionFromToAddresses
+```SOLIDITY
+/** @dev Returns the from (customer) and to (business) addresses of a subscription.
+  * @param _subscription is the identifier of the customer's subscription with its relevant details.
+  * @return from is the address where recurring payments are deducted (customer).
+  * @return to is the address where recurring payments are sent (business).
+*/
+function getSubscriptionFromToAddresses(bytes32 _subscription)
+    public
+    view
+    returns (address from, address to);
+```
+
+###### getSubscriptionOwnerBalance
+```SOLIDITY
+/** @dev Gets token balance of the subscription owners address.
+  * @param _subscription is the identifier of the customer's subscription with its relevant details.
+  * @return balance the token balance of the address where the recurring payments will be deducted (the customer).
+*/
+function getSubscriptionOwnerBalance(bytes32 _subscription)
+    public
+    view
+    returns (uint balance);
+```
+
+###### getAmountDueFromSubscription
+```SOLIDITY
+/** @dev Gets the token amount due from the subscription.
+  * @param _subscription is the identifier of the customer's subscription with its relevant details.
+  * @return amount is token amount due from the subscription.
+*/
+function getAmountDueFromSubscription(bytes32 _subscription)
+    public
+    view
+    returns (uint amount);
+```
+
+###### getSubscriptionFee
+```SOLIDITY
+/** @dev Gets the subscription fee.
+  * @param _subscription is the identifier of the customer's subscription with its relevant details.
+  * @return fee is the subscription fee.
+*/
+function getSubscriptionFee(bytes32 _subscription)
+    public
+    view
+    returns (uint fee);
+```
+
```

Comment:
is it worth adding a `getSubscription()` method to minimize the amount of RPC calls that need to be made to learn about a subscription?